python/doc/examples/meta_modeling/fields_metamodels/plot_fieldfunction_metamodel.py

```diff
@@ -6,14 +6,14 @@
 #
 # In this example we are going to create a metamodel of a field function following these steps:
 #
-# - Creation of a field model over an 1-d mesh
-# - Creation of a Gaussian process
-# - Karhunen-Loeve decomposition of a process with known covariance function
-# - Karhunen-Loeve decomposition of a process with known trajectories
-# - Projection of Fields
-# - Functional chaos decomposition between the coefficients of the input and output processes
-# - Build a metamodel of the whole field model
-# - Validate the metamodel
+# Step 1: Creation of a field model over an 1-d mesh.
```

Comment:
Maybe use an enumerated list: https://docutils.sourceforge.io/docs/user/rst/quickref.html#enumerated-lists

I think the rendering would be better.